gossip/src/cluster_info.rs

```diff
@@ -1398,22 +1393,28 @@ impl ClusterInfo {
         } else {
             peers.iter().map(|peer| &peer.tvu).collect()
         };
-        let mut sent = 0;
-        while sent < dests.len() {
-            match multicast(s, &packet.data[..packet.meta.size], &dests[sent..]) {
-                Ok(n) => sent += n,
-                Err(e) => {
-                    inc_new_counter_error!(
-                        "cluster_info-retransmit-send_to_error",
-                        dests.len() - sent,
-                        1
-                    );
-                    error!("retransmit result {:?}", e);
-                    return Err(GossipError::Io(e));
+        let mut dests = &dests[..];
+        let data = &packet.data[..packet.meta.size];
+        while !dests.is_empty() {
+            match multicast(s, data, dests) {
+                Ok(n) => dests = &dests[n..],
+                Err(err) => {
```

Comment:
Out of curiosity, what types of errors was this multicast failing with?

Reply:
I linked to the discord chat in the description:

It says: 
https://discord.com/channels/428295358100013066/689412830075551748/849530845052403733
```
ERROR solana_core::cluster_info] retransmit result Os { code: 1, kind: PermissionDenied, message: "Operation not permitted" }
```